install-review-tools.sh

```diff
@@ -4,6 +4,6 @@ HOME=/home/ubuntu
 
 sudo apt-get update -qqy
 sudo apt-get install -qy unzip build-essential charm-tools python-dev python-pip python-virtualenv rsync
-sudo pip install bundletester flake8
+sudo pip install bundletester flake8 pyyaml --upgrade
```

Comment:
you could use the "-U" flag to update to the latest version.

Just a minor comment, not needed for this merge.